client/src/components/LoginSession/LoginSession.js

```diff
@@ -119,19 +105,18 @@ function LoginSession(props) {
             , {lastActiveStr}
           </span>
         </p>
-        {props.IsCurrent &&
         <p>
-          <strong className={'text-success'}>
-            {currentStr}
-          </strong>
+          {props.IsCurrent &&
+            <strong className={'text-success'}>{currentStr}</strong>
+          }
+          {!props.IsCurrent && <a
+            role={'button'}
+            tabIndex={'0'}
+            className={'login-session__logout'}
+            onClick={() => attemptLogOut()}
+            onKeyDown={(event) => { if (enterKeyPressed(event)) { attemptLogOut(); } }}
+          >{logOutStr}</a>}
```

Comment:
You only need the curly brackets when you want to pass a variable as a prop. When you are passing a plain string, just use regular quotes.

```suggestion
          {!props.IsCurrent && <a
            role="button"
            tabIndex="0"
            className="login-session__logout"
            onClick={attemptLogOut}
            onKeyDown={(event) => { if (enterKeyPressed(event)) { attemptLogOut(); } }}
          >{logOutStr}</a>}
```

Reply:
Updated